openslides/agenda/models.py

```diff
@@ -163,7 +163,7 @@ def get_title(self):
         Return the title of this item.
         """
         if not self.content_object:
-            agenda_title = self.title
+            agenda_title = self.title or ""
```

Comment:
This is not the case for items which are related item (see line 165) but for items, where the title is None. This should never happen, but if there are an item without a content_object and without a title in the database, then there should be no error anyway.